### DISCUSSION TITLE
LocationHandler - Bearing

### DISCUSSION THREAD

**Discussion by CodeK1988** [mapsforge/vtm#1145] [Q&A] [answered]
hi mate, regarding the question of which value LocationHandler bearing should take.
Bearing has two values, one comes from the position and the other comes from the bearing angle of the compass. 
The code is as follows
`
location.getBearing()  and onSensorChanged
`
How does the Bearing of LocationHandler get its value when I move?

@devemux86 @mapsforge

**Comment by devemux86** [comment, marked as answer]
@CodeK1988 See the location examples in the [Samples](https://github.com/mapsforge/vtm/tree/master/vtm-android-example).

You can set a callback and provide the bearing from any source.
